### ISSUE TITLE
Publish a new release

### ISSUE THREAD

**Issue by gilles-degols** [fusepy/fusepy#118]
The fusepy is a very good library, but the latest release has been generated ~2 years ago. Many improvements have been done since then, but unfortunately there is no easy way to integrate them in other products with an easy "pip install fusepy".

**Comment by paulo-raca** [comment]
Completely agree, pip should be up-to-date.
In the mean there is an easy  way: `pip install git+https://github.com/fusepy/fusepy.git`

**Comment by ihaveamac** [comment]
You can also install using the archive `pip install https://github.com/fusepy/fusepy/archive/master.zip`, since I think `git+http://...` requires git to be installed, while this doesn't.

Or what I do now is just include fuse.py in my project (primarily for the Windows support), and watch the repo for a new release, because adding non-pypi packages as dependencies is really annoying.

**Comment by gilles-degols** [comment]
@paulo-raca Thank you for the reminder about pip install from git. To be sure to have a consistent version at every build, I prefer to specify the git commit: 
pip install git+https://github.com/fusepy/fusepy.git@b5f87a185

Anyway, I hope a new version will be officially published with an easy-to-remember version (rather than an ugly commit id)

**Comment by rsignell-usgs** [comment]
I see that fusepy is now also available on the awesome conda-forge channel, but of course it's the two-year-old 2.0.4 version: https://anaconda.org/conda-forge/fusepy/files

Even more reason to issue a new release!  
@eestrada, any reason not to?

**Comment by eestrada** [comment]
Hi everyone. I agree we should officially put out some new releases now that a number of changes have been merged. To answer @rsignell-usgs directly, the credentials for pypi are still held exclusively by @terencehonles  for the time being. so there isn't much I can do to push new versions there.

I can, however, tag commits in the git repo so that instead of doing something like `pip install git+https://github.com/fusepy/fusepy.git@b5f87a185` you can do something a little cleaner like `pip install git+https://github.com/fusepy/fusepy.git@v2.1.0`. I'll see if @terencehonles replies to this in the next day or two. If not, I'll start pushing some new tags so that people can anchor their dependencies to something.

**Comment by ocefpaf** [comment]
@eestrada tags or, even better, a GitHub release would really help. Thanks!

**Comment by rsignell-usgs** [comment]
Yes, if we have a new github release we can cleanly update the conda-forge fusepy package, even if pypi isn't updated.

**Comment by jjardon** [comment]
@eestrada any plan to make a new tag anytime soon?

**Comment by pink-mist** [comment]
https://pypi.org/project/fusepy/ shows the version as 3.0.1, but no corresponding tag here on github? What's going on?
